wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WebUtilities.java

```diff
@@ -525,9 +525,16 @@ public static UIContext findClosestContext(final String id)
      * and all its bean-bound children.
      *
      * @param component the component whose contents need to be copied to the bean.
+     * @param ignoreVisible - whether to ignore visible components.
      */
-    public static void updateBeanValue(final WComponent component)
+    public static void updateBeanValue(final WComponent component, final boolean ignoreVisible)

```

Comment:
I think the "ignoreVisible" might be confusing. Is it meant to be ignoring "invisible"?

There is bit of a convention in WComponents of using "visibleOnly" as the flag. (e.g. the getComponentById method in this class, or look at TreeUtil). So would be good to switch the flag to "visibleOnly" and then pass "true" to only process the visible components and "false" to process all.